.github/workflows/linux.yml

```diff
@@ -0,0 +1,64 @@
+---
+name: linux
```

Comment:
Instead of adding this file, then you can expand the matrix in:

``.github/workflows/cijoe_packages.yml``

example:

```
    strategy:
      fail-fast: false
      matrix:
        usage_example: ['fio', 'linux', 'system_imaging', 'qemu.build']
        python-version: ['3.12']
```